### ISSUE TITLE
fail make test

### ISSUE THREAD

**Issue by murasesyuka** [mattn/mruby-thread#42]
Merge pull request #40 is fail "make test".

env : linux x86_64, gcc 5.3.1

``` shell
$ make test
...
...
...
>>> Test test <<<
mrbtest - Embeddable Ruby Test

.........................................................................................................................................................................................................................................................................................................................................XF...............................................................................................................................................................................................................................................................................................................................................................?........................................................................................................................................................................................................................................................
NoMethodError: Thread returns Array => undefined method '==' for #<#<Class:0x1faf0c0>:0x1fa8dc0> (mrbgems: mruby-thread)
Fail: Thread returns Hash (mrbgems: mruby-thread)
Skip: Module#prepend super in alias  super does not currently work in aliased methods
Total: 930
   OK: 928
   KO: 1
Crash: 1
 Time: 0.42 seconds
rake aborted!
Command Failed: ["build/test/bin/mrbtest"]
Rakefile:119:in `block (2 levels) in <top (required)>'
Makefile:12: recipe for target 'test' failed
make: *** [test] Error 1
```

**Comment by mattn** [comment]
Hmm, I get below.
```
>>> Test test <<<
mrbtest - Embeddable Ruby Test

..................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................?................................................................................................................................................................................
Skip: Module#prepend super in alias  super does not currently work in aliased methods
Total: 930
   OK: 930
   KO: 0
Crash: 0
 Time: 0.71 seconds

.............
Total: 13
   OK: 13
   KO: 0
Crash: 0
 Time: 1.67 seconds
```

**Comment by murasesyuka** [comment]
I retried from "git clone" of mruby. and it same error happen. 

Do you tested to it with Windows?

**Comment by mattn** [comment]
>Do you tested to it with Windows?

Yes, I'll try this on linux too.

**Comment by mattn** [comment]
should be fixed by #36
